app-drawer/app-drawer.html

```diff
@@ -137,7 +139,8 @@
       transition: opacity 0.2s ease;
 
       -webkit-transform: translateZ(0);
-      transform:  translateZ(0);
+      transform: translateZ(0);
+      will-change: opacity;
```

Comment:
I think we could remove `transform: translateZ(0);` hack since we are now using `will-change` to notify the browser about the `opacity` change.  We still need `-webkit-transform: translateZ(0);` for Safari 9 or earlier.